inabox/tests/setup_infra.go

```diff
@@ -169,30 +188,47 @@ func setupLocalInfrastructure(
 		return nil, fmt.Errorf("failed to deploy experiment: %w", err)
 	}
 
-	// Start churner server
+	// Start churner goroutine
 	infra.Logger.Info("Starting churner server")
-	err = StartChurnerForInfrastructure(infra)
+	churnerURL, err := StartChurnerForInfrastructure(infra, anvilRPC)
 	if err != nil {
 		return nil, fmt.Errorf("failed to start churner server: %w", err)
 	}
-	infra.Logger.Info("Churner server started", "port", "32002")
+	infra.Logger.Info("Churner server started", "address", churnerURL)
+
+	// Start operator goroutines
+	infra.Logger.Info("Starting operator goroutines")
+	err = StartOperatorsForInfrastructure(infra, anvilRPC, churnerURL)
```

Comment:
think all binaries should be started in a different fct as infra. they are very different objects